src/x/config/configflag/flag.go

```diff
@@ -58,7 +58,7 @@ func (opts *Options) Register() {
 func (opts *Options) RegisterFlagSet(cmd *flag.FlagSet) {
 	opts.cmd = cmd
 
-	cmd.Var(&opts.ConfigFiles, "f", "Configuration files to load")
+	cmd.Var(&opts.ConfigFiles, "f", "DebugConfiguration files to load")
```

Comment:
hm did this need to change? this is just command line flag param description, don’t think we changed the underlying type?

Reply:
nope. bad refactor in goland